app/controllers/jobseekers/profiles/employments_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Jobseekers::Profiles::EmploymentsController < Jobseekers::ProfilesController
+  def new; end
```

Comment:
```suggestion
```

Not sure if we need this and the `edit` action to be explicitly defined

Reply:
they are not needed, i dunno joe said otherwise yesterday when looking at my task? ill leave them there for the moment